digital_comms/runner.py

```diff
@@ -342,21 +382,23 @@ def run():
             logging.info("Annual scenario adoption rate is %s", annual_adoption_rate)
 
             # get adoption desirability from previous timestep
+            #TODO: this needs to count the premises, not the distributions
```

Comment:
`TODO` here and below - can these be fixed up or created as issues?

Reply:
I have resolved these in the next pull request